### PR TITLE
[Terms Entry] UI UX 

### PR DESCRIPTION
**Pull request by mmhughes**
### Description

- Added the following entries to UI UX topic: Bias in User Research, Card Sorting, Interaction Design, Layout, Problem Statement, Prototype, Research Plan, Research Report, Visual Design Elements, Visual Design Principles,

### Type of Change

- Adding new entries

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:25]: [Proofreading] Remember to put backticks when using code snippets. 
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

*“Would you agree that you are satisfied with the current interface?”* This question implies that the user is satisfied with the interface and could lead to acquiescence bias. <br>    

*“How do you feel about the current interface?”* This phrasing is open-ended, so it does not lead participants to agree with a particular sentiment. </details>
``` 
```

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:41]:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

<p>This survey asked,</p> <q>Which of the following emotions do you feel when at the airport? Please select the options below.</q>  

![Bar Chart that displays the data from the survey question: Which of the following emotions do you feel at the airport?](media/confirmation-bias-chart.png) 

The team’s larger goal is to make a product that boosts positive feelings at the airport. Confirmation bias might mean looking at this data and only reporting findings based on the statistic “57.1 of our participants are excited when traveling at the airport.” While this is true, it ignores the other data and emotions, that 42.9% also feel anxious, confused, and frustrated.  </details> 
```
```

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:53]:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

A researcher runs a concept test to evaluate an early design idea. The participant may believe that the goal is to validate the idea so the company can move on to the development phase. This belief could influence them to try to “help” the researcher. They might provide positive feedback even if they don’t like the idea. </details> 
```
```

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:71]:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

Imagine that 20 survey respondents rated their overall approval of a certain product on a Likert scale of 1 to 5.  

![Likert Scale Bar Chart that displays the data from the survey question: Approval from 1-5](media/framing-effect.png) 

The researcher reports that the average approval rate is between 2 and 3, meaning that users are slightly dissatisfied. Considering that the mean is 2.45, this is technically correct. But it is a disingenuous finding &mdash; only 25% of the users actually rated the product between a 2 and a 3.  

It would be more accurate to report that the majority of users are either extremely dissatisfied or extremely satisfied with the product. By reporting the finding that the approval rate is between 2 and 3, the team might miss the root of the issue that causes disparities in the satisfaction of certain groups of users. </details> 
```
```

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:85]:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

Imagine that a team of four researchers conducts user interviews. Three of the researchers are 25 years old and one researcher is 50 years old. All interview participants are 20-25 years old.  

When reviewing the data, the team noticed a clear trend: participants who talked to the younger researchers shared detailed responses that indicated honesty and openness. By contrast, most participants who spoke with the older researcher shared little detail, and their responses indicated an unwillingness to share openly. In this case, participants may have held a bias against the older researcher, which impacted how they approached the interview. </details> 
```
```

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:97]:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

If a survey asks, “How much time do you typically spend on a popular social media app every day?”, participants may not know the exact amount. Their guess could be inaccurate, or they may report a lower number if they feel guilty about spending too much time on social media. To minimize response bias in this scenario, researchers need a way to obtain more accurate data. Some phones automatically track how much time is spent on each app. Asking participants to reference the data tracked by their phone should lead to more accurate responses. </details> 
```
```

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:111]:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

Imagine that a researcher starts an interview with this question: <q>How often do you call your parents?</q> The participant may feel that they call their parents too frequently or infrequently. As a result, they might adjust their answer to match what they believe is normal or respectable.  

To better lead into this topic, the interview could start with questions that are less personal, such as <q>In general, do you prefer texting or calling?</q> and <q> When you reach out to family members, how do you usually contact them?</q> Researchers should be mindful of topics that could spark social desirability bias. That way, they can carefully structure the study to help participants share openly and honestly. </details> 
```
```

File: content/uiux/concepts/problem-statement/problem-statement.md
Comment (CBID2) [inline content/uiux/concepts/problem-statement/problem-statement.md:14]: [suggestion] I think adding an example of a problem statement would help readers gain a better understanding of how it looks. 
```suggestion
A **problem statement** succinctly describes what problem a design will attempt to solve. It is a standard output of the “define” stage of the [design thinking process](https://www.codecademy.com/resources/docs/uiux/design-thinking).  

### Example

```

File: content/uiux/concepts/research-report/research-report.md
Comment (CBID2) [inline content/uiux/concepts/research-report/research-report.md:13]: [proofreading] Consider renaming the title of this section. It'll help readers gain a better understanding of what is being described. 
```suggestion
## Components of a Research Report
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @mmhughes! :) I gave some tips on how to improve your entry. Other than that, it's off to a great start! 👍🏾 (Note: I'm not a maintainer. Just a contributor who's eager to help! :) )

**Comment by lucwhite** [comment]
Hey @CBID2, thanks for your review!

For the backticks comments, these aren't code snippets but rather HTML formatting within the markdown to put this very lengthy information to make the page less overwhelming to skim over on a first pass. I'm going to reject these suggestions but will tag @SSwiniarski to confirm that these will show up properly!

**Comment by CBID2** [comment]
> Hey @CBID2, thanks for your review!
> 
> For the backticks comments, these aren't code snippets but rather HTML formatting within the markdown to initially hide this very lengthy information to make the page less overwhelming to skim over on a first pass. I will not include these suggestions but will tag @SSwiniarski to confirm that these will show up properly!

I see @mmhughes, but it would be best to show them as code snippets. It'll help readers gain a better understanding of how the code structures influence the design of a website.

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (lucwhite) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:39]: Just want to confirm with review that this is a proper way to link to media content in docs entries.

Applies to all media links below.

**Review by lucwhite — APPROVED** [review]
Looks good to me! Will wait on next reviewer. Per input from @SSwiniarski we may have to remove bold styling for first mentions of new terms in docs.

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (CBID2) [reply content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:39]: Hi @mmhughes! :) According to the [File Hosting section of the style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting), you'd have to upload them to the [/media folder](https://github.com/Codecademy/docs/tree/main/media).

Comment (SSwiniarski) [reply content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:39]: @lucwhite the link should be `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name.       The image files should be in the `/media` folder. (The image links will work once the PR is merged.)

Comment (yangc95) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:39]:
```suggestion
![Bar Chart that displays the data from the survey question: Which of the following emotions do you feel at the airport?](https://raw.githubusercontent.com/Codecademy/docs/main/media/confirmation-bias-chart.png)
```

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (yangc95) [inline content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:67]:
```suggestion
![Likert Scale Bar Chart that displays the data from the survey question: Approval from 1-5](https://raw.githubusercontent.com/Codecademy/docs/main/media/framing-effect.png)
```

File: content/uiux/concepts/card-sorting/card-sorting.md
Comment (yangc95) [inline content/uiux/concepts/card-sorting/card-sorting.md:18]:
```suggestion
![A GIF of someone doing a card sorting activity in Miro. The mouse is moving cards labeled "Platforms", "Heels", "Sandals", and "Sneakers" into a group called "Shoes".](https://raw.githubusercontent.com/Codecademy/docs/main/media/card-sorting.gif)
```

**Comment by lucwhite** [comment]
> I see @mmhughes, but it would be best to show them as code snippets. It'll help readers gain a better understanding of how the code structures influence the design of a website.

@CBID2 This content is not about code or the details/summary tags, it's about user research. If it's in a code snippet, the desired effect from using the details/summary tags will not be shown.

**Comment by CBID2** [comment]
> > I see @mmhughes, but it would be best to show them as code snippets. It'll help readers gain a better understanding of how the code structures influence the design of a website.
> 
> 
> 
> @CBID2 This content is not about code or the details/summary tags, it's about user research. If it's in a code snippet, the desired effect from using the details/summary tags will not be shown.

Ohh I see. Thanks for clarifying.

**Review by yangc95 — APPROVED** [review]
LTGM

File: content/uiux/concepts/research-plan/research-plan.md
Comment (SSwiniarski) [inline content/uiux/concepts/research-plan/research-plan.md:18]: Remove blank lines between elements in the bulleted list.

File: content/uiux/concepts/research-report/research-report.md
Comment (SSwiniarski) [inline content/uiux/concepts/research-report/research-report.md:22]: Remove blank lines between elements in the bulleted list.

File: content/uiux/concepts/visual-design-elements/visual-design-elements.md
Comment (SSwiniarski) [inline content/uiux/concepts/visual-design-elements/visual-design-elements.md:16]: Remove blank lines between elements of the bulleted list.

File: content/uiux/concepts/visual-design-elements/visual-design-elements.md
Comment (SSwiniarski) [inline content/uiux/concepts/visual-design-elements/visual-design-elements.md:40]:
```suggestion
- **A.** The _typography_ in this section uses the same font in different sizes and weights.  
- **B.** The _texture_ made up of tiny dots adds visual interest and helps divide the top of the page from the bottom.
- **C.** The bright purple _color_ directs our attention to the call-to-action buttons.
- **D.** This rectangle is made up of thin _lines._
- **E.** This box has _form_, which means it appears to be three-dimensional.
- **F.** The white rectangles are _shapes_ that serve as containers for the code challenges.
- **G.** The _space_ between the rectangles is consistent, creating symmetry.
```

File: content/uiux/concepts/visual-design-principles/visual-design-principles.md
Comment (SSwiniarski) [inline content/uiux/concepts/visual-design-principles/visual-design-principles.md:16]: Remove blank lines between elements of the bulleted list.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@mmhughes @lucwhite I did a review and it mostly looks good. Primarily I want you to eliminate the blank lines between items on bulleted lists, and to consider breaking the bias entry into a general bias concept with terms underneath for each of the bias examples.

File: content/uiux/concepts/card-sorting/card-sorting.md
Comment (lucwhite) [reply content/uiux/concepts/card-sorting/card-sorting.md:1]: @SSwiniarski I definitely think that we're approaching a critical mass for UX concepts and we should start putting things into terms. I wonder though if that's something better put into issues to avoid overcomplicating this already-large PR. Thoughts?

File: content/uiux/concepts/bias-in-user-research/bias-in-user-research.md
Comment (lucwhite) [reply content/uiux/concepts/bias-in-user-research/bias-in-user-research.md:1]: I said this below, but applies here as well:
> I definitely think that we're approaching a critical mass for UX concepts and we should start putting things into terms. I wonder though if that's something better put into issues to avoid overcomplicating this already-large PR. Thoughts?

File: content/uiux/concepts/research-plan/research-plan.md
Comment (lucwhite) [reply content/uiux/concepts/research-plan/research-plan.md:18]: Removed

File: content/uiux/concepts/research-report/research-report.md
Comment (lucwhite) [reply content/uiux/concepts/research-report/research-report.md:22]: Removed

File: content/uiux/concepts/visual-design-elements/visual-design-elements.md
Comment (lucwhite) [reply content/uiux/concepts/visual-design-elements/visual-design-elements.md:16]: Removed

File: content/uiux/concepts/visual-design-principles/visual-design-principles.md
Comment (lucwhite) [reply content/uiux/concepts/visual-design-principles/visual-design-principles.md:16]: Removed

**Review by SSwiniarski — APPROVED** [review]
Approving so we can merge this.

**Pull request merged**
